cohort-engine/src/test/java/com/ibm/cohort/engine/BaseFhirTest.java

```diff
@@ -38,27 +40,32 @@
 import org.hl7.fhir.r4.model.Resource;
 import org.junit.BeforeClass;
 import org.junit.Rule;
+import org.opencds.cqf.cql.engine.execution.LibraryLoader;
 
+import com.github.dnault.xmlpatch.repackaged.org.apache.commons.io.FilenameUtils;
 import com.github.tomakehurst.wiremock.client.MappingBuilder;
 import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import com.ibm.cohort.engine.translation.CqlTranslationProvider;
+import com.ibm.cohort.engine.translation.InJVMCqlTranslationProvider;
 
 import ca.uhn.fhir.context.FhirContext;
 import ca.uhn.fhir.parser.IParser;
 
 public class BaseFhirTest {
-	
+
 	static int HTTP_PORT = 0;
-	
+
 	@BeforeClass
 	public static void setUpBeforeClass() {
 		// get a random local port for test use
-		try( ServerSocket socket = new ServerSocket(0) ) {;
+		try (ServerSocket socket = new ServerSocket(0)) {
+			;
```

Comment:
Extra `;`?

Reply:
Yeah. This is the second one that was found in this PR. I'm not sure where they are coming from. I'm going to blame my Eclipse formatter until you can prove otherwise. :-) Will fix.